tsdb/compact.go

```diff
@@ -533,10 +537,10 @@ func (c *LeveledCompactor) CompactWithBlockPopulator(dest string, dirs []string,
 		}
 	}
 
-	return uid, errs.Err()
+	return []ulid.ULID{uid}, errs.Err()
```

Comment:
Because there is no block, since there was an error
```suggestion
	return nil, errs.Err()
```